bumble/device.py

```diff
@@ -2393,11 +2393,10 @@ def on_connection(
             # If supported, read which PHY we're connected with before
             # notifying listeners of the new connection.
             if self.host.supports_command(HCI_LE_READ_PHY_COMMAND):
+
                 async def read_phy():
                     result = await self.send_command(
-                        HCI_LE_Read_PHY_Command(
-                            connection_handle=connection_handle
-                        ),
+                        HCI_LE_Read_PHY_Command(connection_handle=connection_handle),
```

Comment:
any logical change here or just different formatting?

Reply:
Yes. Just the formatter. I'm not sure how this line made it un-formatted in the recent formatting PR.

Reply:
It was merged just before, if I remember correctly